### DISCUSSION TITLE
Installation in Docker on synology

### DISCUSSION THREAD

**Discussion by LowRadiance** [netbox-community/netbox-docker#726] [Getting Help]
I'm trying to run netbox on a docker in a synology. Is there any example or guidance on that ?. I guess I missed some setup for directories and the database.

**Comment by iwdinw** [comment]
https://www.christian-knedel.de/post/2021/april/20210425-docker-netbox/

**Reply by zizglu** [reply]
in english https://www.christian-knedel.de/en/post/2021/april/20210425-docker-netbox/
